dcicutils/ecs_utils.py

```diff
@@ -43,3 +43,25 @@ def update_all_services(self, *, cluster_name):
         for service_name in self.list_ecs_services(cluster_name=cluster_name):
             self.update_ecs_service(cluster_name=cluster_name, service_name=service_name)
         return True
+
+    def list_ecs_tasks(self):
+        """ Lists all available ECS task definitions. """
+        return self.client.list_task_definitions().get('taskDefinitionArns', [])
+
+    def run_ecs_task(self, *, cluster_name, task_name, subnet, security_group):
+        """ Runs the given task name on the given cluster. """
```

Comment:
Expand doc string to say what the requirements are for subnet and security group, maybe. Questions I had: Does it create an entity (an ecs container instance?) and give it these attributes on a one-time basis? Is the cluster created or does it have to exist already? I think from the boto3 doc that it exists already? And the network configuration refers to existing resources but you can't run get a network interface without confirming that it's OK to use these existing things? I'm not expecting a book-length doc string, and maybe even the details of how it does this is not needed. But saying these are existing resources, not new ones, is probably good. And also mentioning that this only starts the task but does not wait for a result is helpful. In my view, the AWS operation is badly named but since you're using their name, I won't quibble. They seem to have two different ways to start a task and picked this because 'start' was already taken.